src/components/@common/Modal/Modal.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode, useEffect, MouseEvent } from 'react';
+import * as S from './Modal.style';
+
+const Modal = ({ isOpen, onClose, children }: { isOpen: boolean; onClose: () => void; children: ReactNode }) => {
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'auto';
+    }
+  }, [isOpen]);
```

Comment:
clean up 시에는 고려하지 않아도 되는지 궁금해요

Reply:
```suggestion
 useEffect(() => {
    if (isOpen) document.body.style.overflow = 'hidden';

    return () => {
      document.body.style.overflow = 'visible';
    };
  }, [isOpen]);
```

다음과 같이 바꿔보았는데요! open 상태일 때 overflow hidden으로 바꾸고 clean up시에 원래 overflow default 값인 visible로 설정되도록 변경해보았습니다!